<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioSettings</class>
 <widget class="QWidget" name="AudioSettings">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Recording device:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="kcfg_Device"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Sampling frequency:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="kcfg_SampleRate"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Sample bit depth:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="kcfg_BitDepth"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
